scripts/ci-e2e.sh

```diff
@@ -26,12 +26,12 @@ WORKING_DIR=/opt/metal3-dev-env
 sudo mkdir -p ${WORKING_DIR}
 sudo chown "${USER}":"${USER}" ${WORKING_DIR}
 
-M3_DEV_ENV_REPO="https://github.com/metal3-io/metal3-dev-env.git"
-M3_DEV_ENV_BRANCH=master
+M3_DEV_ENV_REPO="https://github.com/nordix/metal3-dev-env.git"
+M3_DEV_ENV_BRANCH="run-arbitrary-command-wgslr"
```

Comment:
```suggestion
M3_DEV_ENV_REPO="https://github.com/metal3-io/metal3-dev-env.git"
M3_DEV_ENV_BRANCH=master
```
TODO Revert the change before merging

Reply:
is this needed for testing with integration test ?

Reply:
It was needed for checking that e2e tests work. The change can be removed as soon as [m3-dev-env#703](https://github.com/metal3-io/metal3-dev-env/pull/703) gets merged